docs/reference/operator/api.adoc

```diff
@@ -538,10 +539,24 @@ CollectorSpec is spec to define scheduling and resources for a collector
 |======================
 |Property|Type|Description
 
+|claims|array|  *(optional)* Claims lists the names of resources, defined in spec.resourceClaims,
 |limits|object|  *(optional)* Limits describes the maximum amount of compute resources allowed.
 |requests|object|  *(optional)* Requests describes the minimum amount of compute resources required.
 |======================
 
+=== .spec.collection.logs.fluentd.resources.claims[]
+===== Description
+
+=====  Type
+* array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|name|string|  Name must match the name of one entry in pod.spec.resourceClaims of
```

Comment:
Should be: 
`Name must match the name of one entry in pod.spec.resourceClaims of the Pod where this field is used. It makes that resource available inside a container.`

Reply:
... or just remove the trailing "of" if you intended to shorten